app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -95,6 +97,39 @@ trait BucketRoutes extends Authentication
                   }
                 }
               }
+            } ~
+            post {
+              (authenticate & entity(as[Seq[UUID]])) { (user, sceneIds) =>
+                if (sceneIds.length > BULK_OPERATION_MAX_LIMIT) {
+                  complete(StatusCodes.RequestEntityTooLarge)
```

Comment:
Thought about trying to convert this into a route directive, but decided that would probably be over-engineering at this point. Having rate limiting built in to a directive sounds like a neat way to tackle it though.